content/backend/graphql-scala/1-getting-started.md

```diff
@@ -0,0 +1,212 @@
+  "org.slf4j" % "slf4j-nop" % "1.6.6",
+  "com.h2database" % "h2" % "1.4.196",
+
+  "org.scalatest" %% "scalatest" % "3.0.4" % Test
+)
+
+Revolver.settings
+
+```
+
+</Instruction>
+
+As you can see at the end of file, I'm using [Revolver Plugin](https://github.com/spray/sbt-revolver). This plugin is really helpful during development. It watches filesystem, recompiles changed files and after that reloads HTTP server. So we are able to see all the changes without restarting the server manually.
+
+Let's add Revolver to the project:
+
+<Instruction>
+
+Create directory `project`. Inside of it create two files: `build.properties` and `plugins.sbt`
+In first place type `sbt.version=0.13.6` to define that we want to use the latest version of SBT.
+
+In the second of these files add dependency to the plugin:
+
+```scala
+
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+
+```
+
+</Instruction>
+
+### HTTP Server
+
+Time to create HTTP Server
+
+<Instruction>
+
+Create `Server.scala` file which will be our HTTP server and entry point for the application.
+Use content as follows:
+
+```scala
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.server.Directives._
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+
+import scala.concurrent.Await
+import scala.language.postfixOps
+
+//1
+object Server extends App {
+
+  //2
+  val PORT = 8080
+
+  implicit val actorSystem = ActorSystem("graphql-server")
+  implicit val materializer = ActorMaterializer()
+
+  import actorSystem.dispatcher
+  import scala.concurrent.duration._
+
+  scala.sys.addShutdownHook(() -> shutdown())
+
+  //3
+  val route: Route = {
+    complete("Hello GrahpQL Scala!!!")
+  }
+
+  Http().bindAndHandle(route, "0.0.0.0", PORT)
+  println(s"open a browser with URL: http://localhost:$PORT")
+
+
+  def shutdown(): Unit = {
+    actorSystem.terminate()
+    Await.result(actorSystem.whenTerminated, 30 seconds)
+  }
+}
+```
+
+</Instruction>
+
+Our server extends an `App` trait so SBT can find it and run when you'll use `sbt run` command. When there are more such files in your project, SBT will ask you which one you want to run.
+
+At point 2, there is defined port number we want to use, choose what you want if proposed `8080` doesn't work for you. The main point of the file is `val route` definitions. For now, in all cases, server responds with simple text to prove it's working, but no worries, you will change this value in the near future.
+
+What is worth pointing out here: In our example I use [Spray JSON](https://github.com/spray/spray-json) library for marshalling and unmarshalling JSON objects, but it isn't obligatory for you. You can use whatever JSON library you want, but in such case you have to change dependencies. [On this page](http://sangria-graphql.org/download/) you can find what JSON libraries Sagria can play with.
```

Comment:
`, but in such case you have to change dependencies` I would drop it - obvious truth is obvious :)